debian/jitsi-meet-web-config.templates

```diff
@@ -4,7 +4,7 @@ __Choices: Generate a new self-signed certificate (You will later get a chance t
 _Description: SSL certificate for the Jitsi Meet instance
  Jitsi Meet is best to be set up with an SSL certificate.
  Having no certificate, a self-signed one will be generated.
- By choosing self-signed you will later have a chance to install Let’s Encrypt certificates.
```

Comment:
I found this line helpful when I installed jitsi, I would keep it.

Reply:
We will drop this entirely in the next PR. I will revert it for now.